fe/src/plugins/ethers.ts

```diff
@@ -54,7 +55,13 @@ export default defineNuxtPlugin(async () => {
 
   smartContract?.on("DonationSent", (sender) => {
     if (sender.toLowerCase() === ethereum.selectedAddress.toLowerCase()) {
-      toast.success(`Fund successfully send!`);
+      if (!isListenerExecuted) {
```

Comment:
To assignee and code reviewer : Pls check another way to avoid duplicate toast if possible